runtime/lib/runtime.cpp

```diff
@@ -245,4 +247,167 @@ void wait(Event event) {
   throw std::runtime_error("runtime is not enabled");
 }
 
+std::vector<float> Tensor::getData() {
+#if defined(TT_RUNTIME_ENABLE_TTNN)
+  return std::vector<float>(static_cast<float *>(this->data.get()),
+                            static_cast<float *>(this->data.get()) +
+                                this->volume);
+#endif
+
+#if defined(TT_RUNTIME_ENABLE_TTMETAL)
+  LOG_WARNING("Getting data for ttmetal tensor is not enabled yet");
+  return {};
+#endif
+}
+
+Tensor OpContext::getOpOutputTensor(CallbackContext context) {
```

Comment:
This function should probably be moved outside of OpContext and added to runtime.cpp and take an OpContext instead to align with how we're implementing the other APIs. Also we should check the current runtime here as well, see `getCurrentRuntime`